### DISCUSSION TITLE
💻【Mac 相关】从零开始配置高效 Mac 开发环境

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#82] [Announcements]
https://imageslr.github.io/2020/03/19/mac-initialization.html 

 Mac 是大多数程序员的主力机器。如今各个互联网公司都会给员工配备 Mac 电脑，而拿到新电脑不免要鼓捣一番，安装各种环境、插件、软件等，以让自己用着更顺手。

**Comment by yangsoon** [comment]
给博主加2个我用起来感觉不错的软件

1. [轻量级卸载工具 appcleaner](https://sspai.com/post/43403)
2. [mos平滑鼠标滚动工具](https://github.com/Caldis/Mos)

**Comment by imageslr** [comment]
@yangsoon 感谢推荐，第二个工具真好用

**Comment by yangsoon** [comment]
@imageslr 那我再推荐几个嘻嘻
1. [fork git客户端 当watch的项目有新的更新会通知用户](https://git-fork.com/)
2. [手绘风  画图工具](https://excalidraw.com/)
3. [forklift 第三方文件管理工具 功能强大美观](https://sspai.com/post/40554)

**Comment by imageslr** [comment]
@yangsoon 我在很多博客上看到的手绘风图片，原来是第二个软件画的，第三个也很赞。感谢精品内容分享！

**Comment by yangsoon** [comment]
> @imageslr
> @yangsoon 我在很多博客上看到的手绘风图片，原来是第二个软件画的，第三个也很赞。感谢精品内容分享！

第二个虽然是个网站 但是支持了chrome导出应用 所以也可以像使用软件一样使用

**Comment by imageslr** [comment]
绘图工具：[draw.io](https://github.com/jgraph/drawio)

**Comment by imageslr** [comment]
附件里下载的压缩包的解压密码就是压缩包名里的网址，如 `xclient.info`。

**Comment by imageslr** [comment]
一个很简洁轻巧的日历工具：https://www.mowglii.com/itsycal/

**Comment by airmelt** [comment]
qlcolorcode and qlimagesize not working

**Comment by imageslr** [comment]
@airmelt Thanks for pointing it out. It may be because these two plugins are not compatible with the newest system, and they don't work for me either.

**Comment by jingxuantju** [comment]
百度云附件好像已经不能访问了

**Reply by imageslr** [reply]
是的，但大部分都可以在 xclient 找到。

**Comment by zhoujiealex** [comment]
推荐一个vim党必须设置的
bash命令行切换成 vim控制模式

set -o vi

这样就有统一的体验了，方便控制修改输入的命令

**Reply by imageslr** [reply]
太棒了！我一直需要这样的功能。我之前一直是 `Ctrl + E, Ctrl + X`！

这个命令输入完成以后，要按一下 ESC，进入 vim normal 模式。

我来更新到这里，感谢建议：https://imageslr.com/2021/vim
